projects/packages/account-protection/src/class-settings.php

```diff
@@ -30,15 +30,31 @@ public function __construct( ?Account_Protection $account_protection = null ) {
 	/**
 	 * Get account protection settings.
 	 *
-	 * @return array
+	 * @return array Settings.
 	 */
-	public function get() {
+	public function get(): array {
 		$settings = array(
 			'isEnabled'                    => $this->account_protection->is_enabled(),
 			'isSupported'                  => $this->account_protection->is_supported_environment(),
 			'hasUnsupportedJetpackVersion' => $this->account_protection->has_unsupported_jetpack_version(),
+			'config'                       => $this->get_config(),
 		);
 
 		return $settings;
 	}
+
+	/**
+	 * Get account protection config.
+	 *
+	 * @return array Config settings.
+	 */
+	public function get_config(): array {
+		$advanced_options = $this->account_protection->environment_supports_advanced_options();
+
+		return array(
+			'supports_advanced_options'                    => $advanced_options,
```

Comment:
This is not a configurable setting, so I think it makes sense to move this out of the `config` setting object being defined here.